sections/06_discussion.md

```diff
@@ -2,57 +2,78 @@
 
 Despite the disparate types of data and scientific goals in the learning tasks
 covered above, several challenges are broadly important for deep learning in the
-biomedical domain.  Here we examine these factors that may impede further
+biomedical domain. Here we examine these factors that may impede further
 progress, ask what steps have already been taken to overcome them, and suggest
 future research directions.
 
-### Evaluation
-
-There are unique challenges to evaluating deep learning predictions in the
-biomedical domain. We focus on TF binding prediction as a representative task to
-illustrate some of these issues. The human genome has 3 billion base pairs, and
-only a small fraction of them are implicated in specific biochemical activities.
-As a result, classification of genomic regions based on their biochemical
-activity results in highly imbalanced classification.  Class imbalance also
-arises in other problems we review, such as virtual screening for drug
-discovery. What are appropriate evaluation metrics that account for the label
-imbalance? The classification labels are formulated based on continuous value
-experimental signals. Practitioners must determine an appropriate procedure for
-formulating binary classification labels based on these signals. In addition,
-the experimental signals are only partially reproducible across experimental
-replicates.  An appropriate upper bound for classification performance must
-account for the experimental reproducibility.
+### General Modeling and Evaluation Considerations
+
+There are challenges with fitting and evaluating deep learning predictions in
+the biomedical domain. To some extent, these difficulties are common to all
+machine learning methods applied to biomedical data. However, the utilization of
+large datasets with high dimensional feature sets for deep learning can
+exacerbate these issues.
+
+Many of the variations on deep learning models described in this review reflect
+design choices informed by foundational machine learning tradeoffs -
+specifically, tuning the flexibility of a model to maximize predictivity for a
+particular use case. In the discussion of patient categorization, we introduced
```

Comment:
These topics are covered below, especially transfer learning.  Should that be the reference instead?  We touch on dropout in the intro but don't really explain it.